### DISCUSSION TITLE
Back Button Behavior: Return to Previous App When First Page in History is Reached

### DISCUSSION THREAD

**Discussion by KHNRV** [plateaukao/einkbro#395] [Ideas]
First of all, thank you for the amazing work on EinkBro! I’m really enjoying using the app.

One of the main ways I use EinkBro is to open news articles from apps like Feeder or Feedly to enhance my reading experience. However, I encountered a potential friction point in the user experience.

### Current Behavior:

When I finish reading an article and press the back button, instead of returning to the previous app (e.g., Feeder or Feedly), I’m prompted with a “No previous page!” message.

### Feature Request:

Could you please consider adding an option that allows the back button to return the user to the previous app after finishing an article? If this feature is already available, I’d appreciate guidance on how to enable it.

Thanks again for your excellent work on this app!

**Comment by plateaukao** [comment]
@KHNRV 
It's configureable in Settings > Behavior > Back Key Behavior.
Check this item, and it'll be what you expected.
